CONTRIBUTING.md

```diff
@@ -36,14 +39,20 @@ Building
 - Building `turbo` CLI: In `cli` run `make turbo`
 - Using `turbo` to build `turbo` CLI: `./turbow.js`
 
-### Testing Turborepo
+### Running Turborepo Tests
 
-From the `cli/` directory, you can
+From the root directory, you can
 
-- run smoke tests with `make e2e`
-- run unit tests with `make test-go`
+- run unit tests with `pnpm run --filter=cli test`
+- run integration tests with `pnpm run --filter=cli integration-tests`
+- run e2e tests with `pnpm run --filter=cli e2e`
 
-To run a single test, you can run `go test ./[path/to/package/]`. See more [in the Go docs](https://pkg.go.dev/cmd/go#hdr-Test_packages).
+To run a single Go test, you can run `go test ./[path/to/package/]`. See more [in the Go docs](https://pkg.go.dev/cmd/go#hdr-Test_packages).
+
+#### Rust Tests
+
+You can run `cargo test -p turborepo-lib`, or [install nextest](https://nexte.st/book/pre-built-binaries.html)
```

Comment:
nextest runs faster, I think it should be recommended first and `cargo test` as the backup option